### DISCUSSION TITLE
Instancing and communicating between .gd and .py file

### DISCUSSION THREAD

**Discussion by DisDoh** [touilleMan/godot-python#265] [General]
Hi,

Is there a way to instanciate a python class in gdscript file and then use the methods of the instanciated object?

I could do:

`

	var ffsnn = load("res://PythonScript.py")
	var ffsnnObject = ffsnn.new(var1,var2,...)

`
But could not do:

`ffsnnObject.function(var1,var2)`

**Comment by TsiamDev** [comment]
Can you elaborate on this?
